lib/App/GitGot/Command/clone.pm

```diff
@@ -61,7 +62,10 @@ sub _execute {
   $new_entry->{tags} = $tags if $tags;
 
   say "Cloning into '$path'..." unless $self->quiet;
-  $new_entry->clone( $repo , $path );
+  $new_entry->clone(
+      ({ recursive => $self->opt->recursive }) x !!$self->opt->recursive,
```

Comment:
This line leaves me a bit frowny, it feels excessively clever. I haven't tested it, but I think simply passing `{ recursive => $self->opt->recursive }` should DTRT?

Reply:
I was playing it safe and just not passing the argument if there was no need, but sure.